src/graph/sampler.cc

```diff
@@ -320,8 +320,10 @@ NodeFlow ConstructNodeFlow(std::vector<dgl_id_t> neighbor_list,
       dgl_id_t dst_id = sub_vers->at(i).first;
       assert(dst_id == neigh_pos->at(i).id);
       size_t pos = neigh_pos->at(i).pos;
-      CHECK_LT(pos, neighbor_list.size());
+      CHECK_LE(pos, neighbor_list.size());
       size_t num_edges = neigh_pos->at(i).num_edges;
+      if (neighbor_list.empty())
+        CHECK(num_edges == 0);
```

Comment:
If you really like having no brace for one line of statement, put them in one line.

Reply:
this is simply a programming habit. i have been doing this for many years. it's not likely to change. i think this kind of variance is ok as long as it meets the google coding style.

Reply:
My habit (largely follows [Linux](https://www.kernel.org/doc/html/v4.18/_sources/process/coding-style.rst.txt)):

```C
/* good */
if (condition)
    one-line statement;

/* good */
if (condition) {
    one-line statement;
}

/* no */
if (condition) one-line statement;

/* no */
if (condition)
{
    one-line statement;
}

/* no-no-no */
if (condition) one-line statement;
    one-line statement that is always executed;
```